CountItems/CountManager.js

```diff
@@ -167,7 +167,17 @@ class CountManager {
         }));
         this.store.buckets += bucketCount;
         this.store.bucketList = this.store.bucketList.concat(transformedInfos);
-        this.q.push(bucketInfos);
+        bucketInfos.forEach(bucketInfo => {
+            this.q.push({bucketInfo}, err => {
+                if (err) {
+                    this.log.error('error processing bucket', {
+                        error: err,
+                        method: 'CountManager::addWork',
+                    });
+                }
+            }
```

Comment:
should we not just add this log in the "global" error handler, instead of adding one hadnler per task?

```
this.q.error(err => {
    this.log.error('error processing bucket', {
        error: err,
        method: 'CountManager::addWork',
    });
    this.q.pause();
    this.q.kill();
    return process.nextTick(onceCB, err);
});
```